src/main/app/src/assets/i18n/en.json

```diff
@@ -111,6 +111,7 @@
   "gegevens.overig": "Other information",
   "gegevens.humantasks": "Task information",
   "gegevens.acties": "Action information",
+  "gegevens.toekennen": "Allocation information",
```

Comment:
Allocate betekent iets anders, de lokatie die je in dat woord ziet staat er neit voor niets.

Reply:
Opgelost in nieuwe commit.